requirements.txt

```diff
@@ -3,6 +3,6 @@ pytest==5.0
 pytest-selenium
 virtualenv
 Appium-Python-Client
-altunityrunner==1.5.7
+AltTester-Driver==1.8.0
 assertpy
 pytest-html
```

Comment:
We don't use `pytest-html`, `assertpy`, `pytest-selenium` or `selenium`, so you can remove them. 

Also there is no need to lock `pytest` to version 5.0, it's better to always install the latest version of `pytest`.